tests/publiccloud/prepare_tools.pm

```diff
@@ -83,7 +83,13 @@ sub run {
     my $azure_error = '/tmp/azure_error';
     record_info('Azure', script_output('az -v 2>' . $azure_error));
     assert_script_run('cat ' . $azure_error);
-    assert_script_run('test ! -s ' . $azure_error);
+    if (script_run('test -s ' . $azure_error)) {
+        die("Unexpected error in azure-cli") unless validate_script_output("cat $azure_error", m/Please let us know how we are doing .* and let us know if you're interested in trying out our newest features .*/);
```

Comment:
hehe that is a really funny error :)

Reply:
yes, Azure CLI as usual causing pain for us :)